src/main/java/org/jitsi/videobridge/util/PartitionedByteBufferPool.java

```diff
@@ -79,12 +74,15 @@
      */
     private boolean enableStatistics = false;
 
+    private final int defaultSize;
+
     /**
      * Initializes a new {@link PartitionedByteBufferPool} instance with
      * a given initial size for each partition.
      */
-    PartitionedByteBufferPool()
+    PartitionedByteBufferPool(int defaultSize)
```

Comment:
had to look for the usage to figure out this is the default buffer size, it's unclear from the name and the comment.  can you change it to `defaultBufferSize`?